py3status/modules/netdata.py

```diff
@@ -59,12 +59,29 @@ class Py3status:
     low_traffic = 400
     med_speed = 60
     med_traffic = 700
-    nic = 'eth0'
+    nic = None
 
     def __init__(self):
         self.old_transmitted = 0
         self.old_received = 0
 
+    def post_config_hook(self):
+        if self.nic is None:
+            """
+            Get default gateway directly from /proc.
+            """
```

Comment:
what's wrong with
`# Get default gateway directly from /proc.` that is the pythonic appoach.  literals are only used in this way for docstrings and are not really comments but documentation.